packages/braintree-integration/src/braintree-paypal/braintree-paypal-payment-strategy.ts

```diff
@@ -247,8 +286,9 @@ export default class BraintreePaypalPaymentStrategy implements PaymentStrategy {
 
             await this.braintreeIntegrationService.getPaypalCheckout(
                 paypalCheckoutConfig,
-                () => {
-                    this.renderPayPalMessages(braintreeOptions?.bannerContainerId);
+                (braintreePaypalCheckout) => {
+                    this.renderPayPalMessages(initializationData);
+                    this.renderPayPalButton(braintreePaypalCheckout);
```

Comment:
It looks a bit complicated.. because when you read the code you can see that `loadPaypalCheckoutInstance` method might be called up to 3 times, and it is not quite clear to me because we don't really render PayPal message or Button in all cases. 

We can put it on hold and refactor in the future, just want to let you know that here is a place, where the code should be updated.